content/tutorial-x-ray-image-processing.ipynb

```diff
@@ -0,0 +1,723 @@
+{
```

Comment:
<p>I think the timeit cells can be put in as markdown, because for CI this may not be a good idea. And the last command also needs to have <code>os.path.join</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Sounds good. Done. Here's how I did it in Markdown @melissawm :

```
_Note on performance:_

- `append`ing the images may no be faster. If you care about performance, you should probably use `np.stack()`, as evidenced when you try to time the code with Python's `timeit`:

    ```python
    %timeit combined_xray_images_1 = np.stack([file1, file2, file3, file4, file5, file6, file7, file8])
    ```

    Example output:

    ```
    1.52 ms ± 49.3 µs per loop (mean ± std. dev. of 7 runs, 1000 loops each)
    ```

    ```python
    %timeit C = [combined_xray_images_2.append(imageio.imread(os.path.join(DIR + '00000011_00'+str(i)+'.png'))) for i in range(8)]
    ```

    Example output:

    ```
    159 ms ± 2.69 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)
    ```
```